src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ruleNode.tsx

```diff
@@ -186,6 +197,20 @@ class RuleNode extends React.Component<Props> {
     return getFieldTypes[fieldConfig.type](name, fieldConfig);
   };
 
+  openModal = (formFields: any) => {
+    this.setState({
+      showModal: true,
+      formFields,
```

Comment:
@ceorourke not quite sure why we are putting `formFields` into state when we can just read the form fields from the `node` prop